pkg/functions/invoke.go

```diff
@@ -19,7 +18,7 @@ const (
 	DefaultInvokeSource      = "/boson/fn"
 	DefaultInvokeType        = "boson.fn"
 	DefaultInvokeContentType = "application/json"
-	DefaultInvokeData        = `{"message":"Hello World"}`
+	DefaultInvokeData        = `{"message": "Hello, World!"}`
```

Comment:
Any reason to change this? The thing is that this change also requires to regenerate the generated doc files, that's why the checks are failing here.

Reply:
Please revert this.